Client/Application/UITestAppDelegate.swift

```diff
@@ -149,6 +149,10 @@ class UITestAppDelegate: AppDelegate, FeatureFlaggable {
             resetApplication()
         }
 
+        if CommandLine.arguments.contains("-disableAnimations") {
```

Comment:
If I set the argument correctly, you don't need this... as it is already implemented here:
https://github.com/mozilla-mobile/firefox-ios/blob/3d1dda8157064f1634c17bed9c41d33998896ffc/Client/Application/UITestAppDelegate.swift#L138

Then you use a launchArgument to disable it as it is done in theory by default unless the set up is overwritten:
https://github.com/mozilla-mobile/firefox-ios/blob/3d1dda8157064f1634c17bed9c41d33998896ffc/Tests/XCUITests/BaseTestCase.swift#L40

Reply:
do not forget to add the argument in the set up so that it works